website_legal_page/README.rst

```diff
@@ -0,0 +1,140 @@
+==================
+Website Legal Page
+==================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fwebsite-lightgray.png?logo=github
+    :target: https://github.com/OCA/website/tree/15.0/website_legal_page
+    :alt: OCA/website
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/website-15-0/website-15-0-website_legal_page
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/186/15.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module was written to provide common legal page needed in any website.
+This legal page must be edited using the website builder.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+Website editor can change easily any text of these pages using website builder.
+
+If you install this module after 'website_sale', it will be merge the content
+of 'website_sale.terms' into a new page.
+
+This module hard-redirect '/shop/terms' to '/legal'.
```

Comment:
I didn't test the full functionality but I just tried it on runbot and I noticed that page `/shop/terms` shows a 404 Page Not Found error.

Page `/terms` seems working instead, but is displaying the standard Odoo terms page and is not redirecting to `/legal`.

The standard Odoo `/terms` page can be changed using website builder, as well as the `/legal` page of this module. 
So either I don't understand what is this module needed for or this module is not needed in Odoo V16 anymore.

Reply:
From my POV these are two different sections... the terms page shows legal terms and conditions for dealing with the company. The legal page is for imprint... 
i can confirm, that the hard redirection from /shop/terms -> terms is not working properly
functionally tested

Reply:
Tested on runbot:

- in the webshop, when viewing a product there's a link to the standard `/terms` page but not to the `/legal` page
- when processing the checkout, the `/legal` page does not appear and there is no enforcing to accept it
- standard Odoo has the checkbox "I agree to the terms & conditions" on checkout, it can be easily enabled with website builder
 
So at the end the standard Odoo covers all the functionalities of this module.

Reply:
@chienandalu @pilarvargas-tecnativa you dealt with this in v15. Can you refresh why is needed?

Reply:
Try the test without installing account and you will see that this terms page does not exist. So I think this module is needed... 

Imagine that you are managing a blog and you want to make accept the legal terms to your users, you will not be able if you don't install account too, but you don't need all the features added on account...

Reply:
> If terms and legal are two different concepts, then why does this module try to redirect from  `/shop/terms` to `/legal`? I'm even more confused now... :grin: 

That's what I meant. For me it would make more sense to the redirect from /shop/terms to /terms.. and have the legal page separate. 
But i guess the approach in this module is different.